### DISCUSSION TITLE
Quickdraw Achievement Badge not showing

### DISCUSSION THREAD

**Discussion by AbdrrahimDahmani** [community/community#60748] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Feature Area

Issues

### Body

I've recently closed an issue from one of my public repositories and 2 pull request from one of my private repositories within 5 min of opening, but I've not gotten my Quickdraw badge even if they said that you should close an issue or a pull request within 5 min of opening, how can I solve it
![image](https://github.com/community/community/assets/80603216/16932be3-478e-4959-be40-0701db1eb297)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Singulariity** [comment]
You probably had to wait a while. I can see you already got it :)

**Reply by AbdrrahimDahmani** [reply]
the thing is I've waited like a day, should I wait more? does it take a time?

**Reply by frantizek** [reply]
It usually does not take that long.
I'd suggest to try again.

**Reply by IvanilsoDaSilva** [reply]
O meu emblema apareceu na mesma hora em que conclui o objetivo.

**Reply by queenofcorgis** [reply]
@IvanilsoDaSilva, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be hiding any comments in languages other than English for now, including this one.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AbdrrahimDahmani ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by AbdrrahimDahmani** [reply]
I'm sorry, I thought it was related to the issues category, still new to the Github community. But thank you for your help!

**Comment by radar-blip** [comment, marked as answer]
You already have the badge, its just hidden from view. 
Check here: https://github.com/AbdrrahimDahmani?tab=achievements and unhide it by clicking the eye icon.

**Reply by AbdrrahimDahmani** [reply]
Oh thank you so much for your reply its is shown now in my profile

**Reply by radar-blip** [reply]
No problem. Happy to help! 👍🏻

**Reply by RenanSX** [reply]
thanks

**Reply by zhangbird050801** [reply]
thanks
